src/components/Community.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { PrimaryBtn, OutlinedBtn2 } from 'styling/Button';
+import { Tagline, Headline5, Error, Headline3 } from '../styling/Text';
+import '../styling/Community.css';
+
+const Community = () => {
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const { register, handleSubmit, formState: { errors } } = useForm();
+
+  const onSubmit = () => {
+    alert('Thank you for joining our community!');
+    setFirstName('');
+    setLastName('');
+    setEmail('');
+    setPassword('');
+  };
+
+  return (
+    <section className="sectionCommunity">
+      <div className="introCommunity">
+        <Tagline>Community</Tagline>
+        <Headline5>Sign up & join our community to get started today</Headline5>
+      </div>
+      <div className="communityCard" style={{ backgroundImage: 'linear-gradient(6deg, #DCD2F1 7%, rgba(255, 255, 255, 0) 50%), url("../images/longstretch.jpg")' }} />
+      <div className="signUpWrapper">
+        <Headline3>Create account</Headline3>
+        <form onSubmit={handleSubmit(onSubmit)}>
+          <div className="nameInput">
+            <fieldset className="name">
+              <legend>First Name</legend>
+              <input {...register('firstName', { required: true, pattern: /^[A-Za-z]+$/i })} onChange={(event) => setFirstName(event.target.value)} value={firstName} />
```

Comment:
Line 36: what is this code doing? I'm unfamiliar with several parts of this and would love to know what's happening here!